### PR TITLE
Bug fix#1627

### PR DESCRIPTION
**Pull request by skrebetsnataliia**
https://github.com/ita-social-projects/GreenCity/issues/1627

fixed position of 'View reply' dropdown for unlogged user

Previous:
image
Result:
image

### CODE REVIEW

**Review by romanovskyroma — APPROVED** [review]

**Review by luchward — APPROVED** [review]

**Review by Benerous — APPROVED** [review]

**Review by vitaliipolitylo — APPROVED** [review]

File: src/app/component/comments/components/comments-list/comments-list.component.html [hunk 1]
```diff
@@ -72,7 +72,8 @@
     <app-like-comment *ngIf="isLoggedIn && (comment.author.id !== +userId)"
                       [comment]="comment"
                       (likesCounter)="changeCounter($event, comment.id, 'likes')"></app-like-comment>
-    <div class="btn-replies" *ngIf="dataType==='comment'">
+    <div class="btn-replies" *ngIf="dataType==='comment'"
```
Comment (mr-pavlenko) [inline src/app/component/comments/components/comments-list/comments-list.component.html:75]: comment - should be constant

**Review by mr-pavlenko — CHANGES_REQUESTED** [review]

**Review by yulkats — APPROVED** [review]

File: src/app/component/comments/components/comments-list/comments-list.component.html (hunk 1 above)
Comment (skrebetsnataliia) [reply src/app/component/comments/components/comments-list/comments-list.component.html]: I have created constant - 'comment'.

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/component/comments/components/comments-list/comments-list.component.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export class CommentsListComponent implements OnInit {
   @Input() public isLoggedIn: boolean;
   @Input() public userId: number;
   @Output() public changedList = new EventEmitter();
+  readonly commentType: string = 'comment';
```
Comment (mr-pavlenko) [inline src/app/component/comments/components/comments-list/comments-list.component.ts:19]: Comment type could be only string, isn't it?

Comment (skrebetsnataliia) [reply src/app/component/comments/components/comments-list/comments-list.component.ts:19]: It should be only string.

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**